### ISSUE TITLE
Implement deleting experiences

### ISSUE THREAD

**Issue by brollin** [SotSF/conjurer#308]
We might want to hold off until we complete user authentication first though

**Comment by milotoor** [comment]
Agreed about user auth, and would also suggest a soft-delete instead of hard
